htdocs/api/common.php

```diff
@@ -36,7 +36,21 @@ function execSuccessfully($command) {
         echo "Execution failed\nCommand: {$command}\nOutput: {$formattedOutput}\nRC: .${rc}";
         http_response_code(500);
         exit();
+    }  
+    return $output;
+}
+
+function execMPDCommand($command) {    
+    $socket = socket_create(AF_INET, SOCK_STREAM, SOL_TCP);
+    $stream = socket_connect($socket,"localhost" ,6600);
+    socket_write($socket, $command, strlen($command));
+    socket_shutdown ($socket,1);
+    $output = array();
+    while ($out = socket_read($socket, 2048)) {
```

Comment:
Is reading 2058 bytes(?) always sufficient?

Reply:
It is in a while loop, so I guess yes. But I need to check with ultralong texts or filenames.

Reply:
Checked with ultralong texts and fixed :) thanks for the remark.